pkgs/development/libraries/apr/default.nix

```diff
@@ -27,6 +27,10 @@ stdenv.mkDerivation rec {
       url = "https://github.com/apache/apr/commit/866e1df66be6704a584feaf5c3d241e3d631d03a.patch";
       sha256 = "0hhm5v5wx985c28dq6d9ngnyqihpsphq4mw7rwylk39k2p90ppcm";
     })
+    (fetchpatch {
+      url = "https://830833.bugs.gentoo.org/attachment.cgi?id=761676";
+      sha256 = "sha256-rsouJp1o7p0d+AJ5KvyhUU79vAJOcVHEuwSEKaCEGa8=";
+    })
   ] ++ lib.optionals stdenv.isDarwin [ ./is-this-a-compiler-bug.patch ];
```

Comment:
Do you know by chance if this patch is still required and could be made unconditional?

Reply:
Yes, it is required, for the reasons discussed here: https://bugs.gentoo.org/show_bug.cgi?id=830833. The cross build will fail without it.

It is currently unconditional, as in, it's always applied.

Reply:
Oh sorry, I think you're talking about `is-this-a-compiler-bug.patch`. I can look into that.

Reply:
Added in a600443bb30e4895bf9d05ba309578be6f8ab951

Reply:
It's 7 years old, and no explanation about why it was required. I tested it out on darwin and the build works fine without the patch.

Reply:
If the build works fine without it I would just remove it.